docs/index.adoc

```diff
@@ -62,16 +62,80 @@ image::diagrams/wallet-lifecycle/wallet-state-transition.png[Wallet State Transi
 
 `Bank` is a central, non-upgradeable contract keeping track of Bitcoin balances. 
 
-`Bridge` is deployed behind a `TransaprentUpgradeableProxy` and can be upgraded
+`Bridge` is deployed behind a `TransparentUpgradeableProxy` and can be upgraded
 by a `ProxyAdmin`, eventually Threshold DAO.
 
 `TBTCVault` upgrade happens in two steps with a 24-hour governance delay. The
 owner of `TBTCVault` can initiate and finalize an upgrade to another vault.
 The new vault receives ownership of TBTC token as well as the entire `TBTCVault`
 balance in the Bank.
 
+=== Bridge, Bank, Vault Separation
+
+The system is decomposed into three separate pieces to separate concerns. The
+<<bridge,Bridge>> is concerned with being a decentralized Bitcoin custody, and
+providing a way to let the <<bank,Bank>> know whenever Bitcoin has successfully
+crossed from Bitcoin to another chain or visa versa.
+
+The <<bank,Bank>> is concerned with keeping track of Bitcoin account balances on
+Ethereum. It tracks when folks send those balances to each other, and when
+folks lock up their balances in <<vault,Vaults>>.
+
+Vaults are authorized smart contracts that can custody a balance and then do
```

Comment:
They do not have to be authorized. Anyone can deploy a vault and have it working with a `Bridge`. The only *authorization* we require is when `Bridge` is allowed to route revealed deposits straight to vaults instead of routing them to the depositor's balance in the Bank.

The reason for it is that a malicious vault, when receiving the balance, could revert the submit sweep proof transaction.